.github/workflows/ci.yml

```
@@ -38,17 +38,26 @@ jobs:
       - name: Build and test all modified packages
         id: test
         # It runs only if there are modified files
+        # We specifically test for "microsoft-windows-terminal.vm" in 'windows-2019' because it requires at least Windows 10 version 1903/OS build 18362
+        # The "test_install.ps1" script will fail if attempting to pass a NULL package list
         if: steps.files.outputs.added_modified_renamed != ''
         run: |
+          $os = "${{ matrix.os }}"
+          $requiredPackages = @("common.vm")
           $packages = "${{ steps.files.outputs.added_modified_renamed }}".Split(" ") | Foreach-Object { (Get-Item $_).Directory.Name }
-          scripts/test/test_install.ps1 "common.vm $packages"
+          if ($os -eq 'windows-2019') {
+            $excludedPackages = @("microsoft-windows-terminal.vm")
+            $packages = $packages | Where-Object { $_ -notin $excludedPackages }
+          }
+          $packages = ($requiredPackages + $packages -join " ").Trim()
+          scripts/test/test_install.ps1 "$packages"
```

`common.vm` is added here manually to force that it is installed first to ensure we test packages with the latest version of `common.vm`. I think the introduction of the `$requiredPackages` is confusing and I would prefer to leave this as it was, as this doesn't need to be changed to exclude the package. Or am I missing something?

```suggestion
          scripts/test/test_install.ps1 "common.vm $packages"
```

---

I can change it back. It made it easier to combine the packages into a single string and gave us room for if there was any other package that might be required to be installed first later down the road, which I felt might be a worthwhile thought.